Source/WTF/wtf/ThreadSafeWeakPtr.h

```diff
@@ -422,8 +441,7 @@ class ThreadSafeWeakPtr {
     template<typename> friend class ThreadSafeWeakOrStrongPtr;
 
     TaggedPtr<T, TaggingTraits> m_objectOfCorrectType;
-    // FIXME: Either remove ThreadSafeWeakPtrControlBlock::m_object as redundant information,
-    // or use CompactRefPtrTuple to reduce sizeof(ThreadSafeWeakPtr) by storing just an offset
+    // FIXME: Use CompactRefPtrTuple to reduce sizeof(ThreadSafeWeakPtr) by storing just an offset
```

Comment:
Please add a radar ink here too.  Out maybe there should can use the same 1 radar link for all these FIXMEs.

Reply:
Done.